django/contrib/staticfiles/handlers.py

```diff
@@ -64,3 +72,35 @@ def __call__(self, environ, start_response):
         if not self._should_handle(get_path_info(environ)):
```

Comment:
This already checks for `_should_handle`. Why is this duplication also in `get_response`? Because if you eliminate the duplication the wsgi and asgi `get_response` are exactly the same and can be moved up into the common class, no?

Reply:
I agree, but considering the code was already like this, I did not want to adjust it without understanding why. Any ideas?

Reply:
I think it can be removed: #11336